README.md

```diff
@@ -19,31 +19,67 @@ _Xamarin Forms provides cross-platform APIs, but this project are only worked on
-- TwoButtonPage : A subclass of Xamarin.Forms.Page has two buttons on the left and right side of the circular screen.
-
-![widgets](doc/design/data/widgets.png)
+The Tizen Wearable CircularUI project provides UI controls to fits circular screen and to interact to user using hardware feature provided from Samsung Gear such as bezel rotation.
+It provides newly created controls for Tizen Wearable.
+
+- [CircleDateTimeSelector](https://samsung.github.io/Tizen.CircularUI/api/Tizen.Wearable.CircularUI.Forms.CircleDateTimeSelector.html) : A View to select date or time that fits in the circular screen. Unlike Xamarin.Forms.DatePicker(TimePicker) it can be shown on the page without popup.
+- [CircleImage](https://samsung.github.io/Tizen.CircularUI/api/Tizen.Wearable.CircularUI.Forms.CircleImage.html) : An Image that crops an image in a circle shape.
+- [CircleProgressBarSurfaceItem](https://samsung.github.io/Tizen.CircularUI/api/Tizen.Wearable.CircularUI.Forms.CircleProgressBarSurfaceItem.html) : A progress control that fits in the circular screen. it can be shown in CirclePage only.
+- [CircleSliderSurfaceItem](https://samsung.github.io/Tizen.CircularUI/api/Tizen.Wearable.CircularUI.Forms.CircleSliderSurfaceItem.html) : A slider control that responds to the bezel action and fits in the circular screen.
+- [CircleStackLayout](https://samsung.github.io/Tizen.CircularUI/api/Tizen.Wearable.CircularUI.Forms.CircleStackLayout.html) : A container to layout children linear in the circular area.
+- [CircleSurfaceEffectBehavior](https://samsung.github.io/Tizen.CircularUI/api/Tizen.Wearable.CircularUI.Forms.CircleSurfaceEffectBehavior.html) : An effect class that allows you to insert views that require CircleSurface.
+- [CircleSurfaceItem](https://samsung.github.io/Tizen.CircularUI/api/Tizen.Wearable.CircularUI.Forms.CircleSurfaceItem.html) : A item class to represent the items in the CircleSurface.
+- [CircleSurfaceView](https://samsung.github.io/Tizen.CircularUI/api/Tizen.Wearable.CircularUI.Forms.CircleSurfaceView.html) : A View that can have several CircleSurfaceItem, not Page.
+- [CircleToolbarItem](https://samsung.github.io/Tizen.CircularUI/api/Tizen.Wearable.CircularUI.Forms.CircleToolbarItem.html) : An item class that can show a toolbar item for fits in the circular screen.
+- [ColorMenuItem](https://samsung.github.io/Tizen.CircularUI/api/Tizen.Wearable.CircularUI.Forms.ColorMenuItem.html) : A item class that allows you to set the background color of the button used in the popup control.
+- [ContentButton](https://samsung.github.io/Tizen.CircularUI/api/Tizen.Wearable.CircularUI.Forms.ContentButton.html) : A Button that allows you to customize the view to show.
+- [ContextPopupEffectBehavior](https://samsung.github.io/Tizen.CircularUI/api/Tizen.Wearable.CircularUI.Forms.ContextPopupEffectBehavior.html) : A behavior to show a small popup  has one or two button sticky with any control.
```

Comment:
A behavior to show a small popup which has one or two buttons. This can be added to any Xamarin.Forms View.